### DISCUSSION TITLE
qiankun 生命周期从创建走到销毁

### DISCUSSION THREAD

**Discussion by AYang518989** [vbenjs/vue-vben-admin#2371] [Q&A]
子应用是vben二次开发的,qiankun官方暂未支持vite,暂时先借助了vite-plugin-qiankun这个插件，主应用加载子应用时qiankun的生命周期从创建一路走到销毁，请问是哪里有问题呢？
![image](https://user-images.githubusercontent.com/86372336/201512971-497293b1-bae4-4bb7-87dd-6bcb7852e108.png)

**Comment by miaomiaoao** [comment]
请问解决了么
